diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter4/NullChecksRuleWarnTest.kt

```diff
@@ -164,7 +168,21 @@ class NullChecksRuleWarnTest : LintTestBase(::NullChecksRule) {
                 |      }
                 | }
                 """.trimMargin(),
-                LintError(2, 10, ruleId, "${Warnings.AVOID_NULL_CHECKS.warnText()} myVar != null", true),
+                LintError(2, 10, ruleId, "${Warnings.AVOID_NULL_CHECKS.warnText()} use '.let/.also/?:/e.t.c' instead of myVar != null", true),
+        )
+    }
+
+    @Test
+    @Tag(WarningNames.AVOID_NULL_CHECKS)
+    fun `require statements - adding `() {
+        lintMethod(
+                """
+                | fun foo0(myVar: String?) {
+                |     require(myVar != null)
+                | }
+                """.trimMargin(),
+                LintError(2, 14, ruleId, Warnings.AVOID_NULL_CHECKS.warnText() +
+                        " use 'requireNotNull()' instead of myVar != null", false),
```

Comment:
Text can be adjusted a little: like `use 'requireNotNull(myVar)' instead of 'require (myVar != null)'`.

Reply:
getting myVar is too complex logic - I would not like to make the code more complex, but the second part is easy enough, added